cmd/soci/commands/ztoc/info.go

```diff
@@ -30,15 +32,23 @@ import (
 )
 
 type Info struct {
-	Version   string     `json:"version"`
-	BuildTool string     `json:"build_tool"`
-	Files     []FileInfo `json:"files"`
+	Version           string             `json:"version"`
+	BuildTool         string             `json:"build_tool"`
+	Size              int64              `json:"size"`
+	SpanSize          compression.Offset `json:"span_size"`
+	MaxSpan           compression.SpanID `json:"max_span_id"`
```

Comment:
Is the max span id something people care about? I think number of spans would be more useful.